appdatabase/migrations/sql/1683627613_accounts_and_keycards_improvements.up.sql

```diff
@@ -0,0 +1,94 @@
+-- Possible `operable` column values:
+-- "no"        // an account is non operable it is not a keycard account and there is no keystore file for it and no keystore file for the address it is derived from
+-- "partially" // an account is partially operable if it is not a keycard account and there is created keystore file for the address it is derived from
+-- "fully"     // an account is fully operable if it is not a keycard account and there is a keystore file for it
+
+-- Adding new tables `keypairs` and `keypairs_accounts`
+CREATE TABLE IF NOT EXISTS keypairs (
+  key_uid VARCHAR PRIMARY KEY NOT NULL CHECK (length(trim(key_uid)) > 0),
+  name VARCHAR NOT NULL DEFAULT "",
+  type VARCHAR NOT NULL DEFAULT "",
+  derived_from VARCHAR NOT NULL DEFAULT "",
+  last_used_derivation_index INT NOT NULL DEFAULT 0,
+  synced_from VARCHAR NOT NULL DEFAULT "", -- keeps an info which device this keypair is added from
+  clock INT NOT NULL DEFAULT 0
+);
+
+CREATE TABLE IF NOT EXISTS keypairs_accounts (
+  address VARCHAR PRIMARY KEY,
+  key_uid VARCHAR,
+  pubkey VARCHAR,
+  path VARCHAR NOT NULL DEFAULT "",
+  name VARCHAR NOT NULL DEFAULT "",
+  color VARCHAR NOT NULL DEFAULT "",
+  emoji VARCHAR NOT NULL DEFAULT "",
+  wallet BOOL NOT NULL DEFAULT FALSE,
+  chat BOOL NOT NULL DEFAULT FALSE,
+  hidden BOOL NOT NULL DEFAULT FALSE,
+  operable VARCHAR NOT NULL DEFAULT "no", -- describes an account's operability (read an explanation at the top of this file)
+  created_at DATETIME NOT NULL,
+  updated_at DATETIME NOT NULL,
+  clock INT NOT NULL DEFAULT 0,
+  FOREIGN KEY(key_uid) REFERENCES keypairs(key_uid) 
+    ON DELETE CASCADE
+);
+
+-- Fulfilling the tables
+INSERT INTO keypairs 
+  SELECT key_uid, keypair_name, "profile", derived_from, last_used_derivation_index, "", clock 
+  FROM accounts 
+  WHERE type != "watch" AND type != "seed" AND type != "key"
```

Comment:
i'm not sure, i thought condition would be:  WHERE type = "" or type = "seed" or type = "key" ? maybe i misunderstood something...

Reply:
This one inserts all accounts referring to a profile keypair.